README.md

```diff
@@ -308,5 +308,7 @@ trait Reader[A] {
 
 Built-in instances are, unsurprisingly, very simple examples to follow:
 ```scala
-implicit val IntParser = new Reader[Int] { def apply(s: String) = Try(s.toInt) }
+implicit val IntParser = safeSummon[Int](s => s.toInt)
```

Comment:
I am not sure the `summon` and `safeSummon` helpers are useful since Scala 2.12 supports [SAM types](https://www.scala-lang.org/news/2.12.0/#lambda-syntax-for-sam-types):

`implicit val IntParser: Reader[Int] = x => Try(x.toInt)`